[CodeStyle][F402] Change the loop variable name which is the same with an imported module

caolonghao opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->
Others
### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->
Others
### Describe
<!-- Describe what this PR does -->
修复 `python\paddle\fluid\tests\unittests\test_auto_parallel_partitioner.py` 中 F402 循环变量使用 import 过的模块名字

### Related Links
- Flake8 tracking issue: #46039 
- 配置文件更新：#46699
- fixes https://github.com/cattidea/paddle-flake8-project/issues/53

caolonghao committed:
Update README.md

caolonghao committed:
Update README.md

caolonghao committed:
Merge branch 'develop' of https://github.com/caolonghao/Paddle into develop

caolonghao committed:
Fix F402 by changing the name of loop variable

SigureMo commented on python/paddle/fluid/tests/unittests/test_auto_parallel_partitioner.py:
```diff
@@ -234,9 +234,9 @@ def distributed_attr_check_for_dist_op(serial_main_prog, dist_main_prog,
     have_dist_attr = True
     for block in dist_main_prog.blocks:
-        for tensor in block.vars.values():
+        for temp_tensor in block.vars.values():
```
`temp_tensor` 可能会让人误以为是一个临时变量，也许使用 `var` 会更合适些？（全局搜 `get_tensor_dist_attr_for_program` 可以发现会有很多 `get_tensor_dist_attr_for_program(var)`）

caolonghao replied:
有道理，后面确实有这样的用法。但在这里似乎也是一个循环中的临时变量？

SigureMo replied:
唔，循环的话，一般不会认为是临时变量的，因为在这个循环里并不是「临时」需要的

caolonghao replied:
有道理，会做出相应修改。

caolonghao committed:
Change variable name
Change variable name 'temp_tensor' to 'var' to avoid confusion, as var
also appears in the same file.

SigureMo reviewed (approved):
LGTM~

SigureMo commented:
PR-CI-Windows 和 PR-CI-Windows-Inference 已经好啦，麻烦 re-run 下之前 PR 的这两个流水线，如果有其他流水线挂的也要 re-run 下（除了 PR-CI-APPROVAL 和 PR-CI-Static-Check）

re-run 后估计大多数 PR 就在过两天就可以 merge 啦～～～

caolonghao commented:
了解~
